app/assets/stylesheets/frontend.less

```diff
@@ -16,34 +16,43 @@ body {
 	font-family: sans-serif;
 }
 
-.section_headings {
-    position: relative;
-    z-index: 1;
-    overflow: hidden;
-    text-align: center;
+h1 {
+	/* Page itself has top padding */
+	margin-top: 0
 }
 
-.section_headings:before, .section_headings:after {
```

Comment:
Might have been better to actually look at how this was used, this is added in https://github.com/hotdoc/hotdoc_bootstrap_theme/blob/master/app/assets/javascript/lines_around_headings.js , which tries to be a bit clever with figuring out what should be considered as a section in the page. Besides you are removing the lines around the main section headings, which would deserve an explanation?